tools/perf_view/ort_perf_view.html

```diff
@@ -0,0 +1,138 @@
+
+        const fileList = event.target.files;
+        console.log(fileList);
+        const reader = new FileReader();
+        reader.addEventListener('load', (event) => {
+          raw_data_in_json = JSON.parse(event.target.result);
+          summarized_cpu = JSON.parse("{}");
+          summarized_gpu = JSON.parse("{}");
+          for (var i = 0; i < raw_data_in_json.length; i++) {
+            var node = raw_data_in_json[i];
+            if (node.cat == null ||node.name == null || node.args == null || node.args.op_name == null) continue;
+            var category = node.cat;
+            if (category == "Node") {
+                if ( summarized_cpu[node.args.op_name] == null) {
+                  summarized_cpu[node.args.op_name] = {all:0,children:[]};
+                }
+                summarized_cpu[node.args.op_name].all += node.dur;
+                summarized_cpu[node.args.op_name].children.push({name: node.name, value:node.dur})
+            } else if (category == "Kernel") {
+                var op_name = node.args.op_name == "" ? "Session" : node.args.op_name;
+                if (summarized_gpu[op_name] == null) {
+                  
+                  summarized_gpu[op_name] = {all:0,children:[]};
+                }
+                summarized_gpu[op_name].all += node.dur;
+                summarized_gpu[op_name].children.push({name: node.name, value:node.dur})
+            }
+          }
+          var array_cpu = JSON.parse("[]");
+          array_cpu.push({Xname:"all", children:[]});
+          for (var op in summarized_cpu) {
+            array_cpu[0].children.push({name:op, children: summarized_cpu[op].children});
+          }
+          var chart_cpu = anychart.treeMap(anychart.data.tree(array_cpu, "as-tree"));
+          chart_cpu.title("CPU perf overview");
+
+          var array_gpu = JSON.parse("[]");
+          array_gpu.push({Xname:"all", children:[]});
+          var has_gpu_info = false;
+          for (var op in summarized_gpu) {
+            array_gpu[0].children.push({name:op, children: summarized_gpu[op].children});
+            has_gpu_info = true;
+          }
+
+          chart_cpu.listen("drillchange", function(e){
+              console.log('cpu drillchange');
+              document.body.style.cursor = "wait";
+              return true;
+          });
+          chart_cpu.listen("chartDraw", function(e){
+              console.log('cpu refreshed');
+              document.body.style.cursor = "default";
+              return true;
+          });
+
+          //chart_cpu.hintDepth(1);
+          //chart_cpu.hintOpacity(0.8); 
+          chart_cpu.container("cpu");
+          chart_cpu.draw();
+          
+           if (has_gpu_info) {
+            document.getElementById("cpu").className = "half_view";
+            var chart_gpu = anychart.treeMap(anychart.data.tree(array_gpu, "as-tree"));
+            chart_gpu.title("GPU perf overview");
+            //chart_gpu.hintDepth(1);
```

Comment:
Same as above.

Reply:
Same as replied above - let's just keep it.